src/dat/gui/GUI.js

```diff
@@ -1165,6 +1172,10 @@ function add(gui, object, property, params) {
   dom.addClass(li, GUI.CLASS_CONTROLLER_ROW);
   if (controller instanceof ColorController) {
     dom.addClass(li, 'color');
+  } else if ( params.liClass ) {
+    dom.addClass(li, params.liClass);
+  } else if ( controller.liClass ) {
+    dom.addClass(li, controller.liClass);
```

Comment:
Are both of these (`params.liClass` and `controller.liClass`) needed? If this `li` will later be set as `controller.__li`, could the controller add the class itself?